src/Harald.WebApi/Infrastructure/Persistence/CapabilityEntityFrameworkRepository.cs

```diff
@@ -34,6 +36,13 @@ public async Task Update(Capability capability)
             _dbContext.Capabilities.Update(capability);
             await _dbContext.SaveChangesAsync();
         }
-        
+
+        public async Task<IEnumerable<Capability>> GetAll()
+        {
+            return await _dbContext
+                .Capabilities
+                .OrderBy(c => c.Name)
+                .ToListAsync();

```

Comment:
Add .AsNoTracking() to bypass tracking in the EF context for better performance

Reply:
`AsNoTracking added`